### PR TITLE
Fix git rpr

### PR DESCRIPTION
**Pull request by emilevauge**
This PR adds an argument to manage upstream correctly: `git rpr pr remote/branch`
It also adds `git rmpr` & `git cpr` commands.

/cc @vdemeester 

Signed-off-by: Emile Vauge <emile@vauge.com>

### CODE REVIEW

File: .github/rpr.sh [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 set -e # stop on error
 
-usage="$(basename "$0") pr -- rebase a Pull Request against current branch"
+usage="$(basename "$0") pr remote/branch -- rebase a Pull Request against a remote branch"
 
-if [ "$#" -ne 1 ]; then
+if [ "$#" -ne 2 ]; then
```
Comment (vdemeester) [inline .github/rpr.sh]: Should we have a default value for the most common case ? (if `$2` is not present, use `upstream/master` for ex 👼 )

**Comment by emilevauge** [comment]
@vdemeester I added 1/. On 2/, is it really needed? The script will fail if the specified remote doesn't exist, right ?

File: .github/cpr.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# git config --global alias.rpr '!sh .github/cpr.sh'
```
Comment (vdemeester) [inline .github/cpr.sh]: This alias is not right 👼

File: .github/rmpr.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+#
+# git config --global alias.rpr '!sh .github/rmpr.sh'
```
Comment (vdemeester) [inline .github/rmpr.sh]: This alias is not right 👼 👼

**Comment by emilevauge** [comment]
@vdemeester 
>It will, but it leaves you in a weird state (not on the same branch when you ran the script initially).
Added a TRAP calling clean.

>Also, shouldn't rpr use cpr and rmpr ?
Indeed, fixed.

**Pull request merged**